acct-user/scponly/scponly-0.ebuild

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="user for chrooted scponly"
+ACCT_USER_ID=239
+ACCT_USER_GROUPS=( scponly )
+ACCT_USER_HOME=/home/scponly//
```

Comment:
`/home` is off limits for the package manager. Presumably some place under `/var/lib` is a better choice.

Also, drop the trailing slashes.

Reply:
> Oh well, a single QA commit to a m-n package makes github think that I would be the right person to do a review here.

Hah, thanks for the review :)

> /home is off limits for the package manager. Presumably some place under /var/lib is a better choice.

Ah, right.  Since submitting this a while back there was the whole thread on gentoo-dev, and I agree with the reasoning.  Will put it under /var/lib.

> Also, drop the trailing slashes.

The double trailing slash is weird, but required by scponly: that is the indicator that "this is the directory to chroot incoming connections into".

Reply:
> Will put it under /var/lib.

Actually, I see acct-user/dhcpcd uses /var/chroot/dhcpcd, and using /var/chroot/ makes sense to me since that is what this is for.  Will use that unless you know already that will be rejected / start another flame war.

Reply:
> Actually, I see acct-user/dhcpcd uses /var/chroot/dhcpcd, and using /var/chroot/ makes sense to me since that is what this is for. 

It's a non-FHS location, but we don't always follow it and in /var we're rather lax. Especially if there's a precedent.

> Will use that unless you know already that will be rejected / start another flame war.

I won't guarantee for that. :)

Reply:
Could you add a comment above this line wrt those slashes? Otherwise, someone could accidentally remove them.

Reply:
> Could you add a comment above this line wrt those slashes? Otherwise, someone could accidentally remove them.

Yup, actually the current version of the patch (since yesterday) contains:

```
# Not a typo.  scponly uses the trailing // to identify the chroot dir.
ACCT_USER_HOME=/var/chroot/scponly//
```